packages/next/export/worker.js

```diff
@@ -92,6 +97,11 @@ process.on(
           // If the path is the root, just use index.html
           htmlFilename = 'index.html'
         }
+
+        if (sprPage) {
+          htmlFilename = htmlFilename.replace(/\.html$/, '.prerender.html')
```

Comment:
What significance (if any) does this extension have?

Reply:
responded on slack